Tests/AppKernel.php

```diff
@@ -37,6 +38,15 @@ public function registerBundles() {
 
 	public function registerContainerConfiguration(LoaderInterface $loader) {
 		$loader->load($this->config);
+
+		if (class_exists('Symfony\Component\Asset\Package')) {
+			// enable assets to avoid fatal error "Call to a member function needsEnvironment() on a non-object in vendor/twig/twig/lib/Twig/Node/Expression/Function.php on line 25" with Symfony 3.0
```

Comment:
I don't need Assetic for the tests, but got an error regarding "asset" being undefined. When trying locally now, it's gone. :unamused: I'll remove this code.